x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/action/user/AuthenticateResponse.java

```diff
@@ -31,7 +31,7 @@ public Authentication authentication() {
     @Override
     public void writeTo(StreamOutput out) throws IOException {
         super.writeTo(out);
-        if (out.getVersion().before(Version.V_6_6_0)) {
+        if (out.getVersion().before(Version.V_7_0_0)) {
```

Comment:
Why is this changed in this PR? This seems unrelated to me.

Reply:
that must have been some debugging remains
removing